common/src/main/java/com/wynntils/services/itemfilter/LevelItemFilter.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.services.itemfilter;
+
+import static java.lang.Integer.parseInt;
+
+import com.wynntils.models.items.WynnItem;
+import com.wynntils.models.items.properties.LeveledItemProperty;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class LevelItemFilter extends ItemFilter {
+    private static final Pattern LEVEL_RANGE_PATTERN = Pattern.compile("^(\\d+)(?:-(\\d+))?$");
+
+    private int minLevel, maxLevel;
```

Comment:
Secondary kind of unwritten code style rule is that fields must be declared in separate lines. Thanks.